src/_data/profiles/seminapini.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "Semina Pini",
+    "jobTitle": "Student Applied IT Science",
+    "company": "PXL University College",
+    "bio": "Someone who is always looking for new challenges. Someone who believes that JAMStack is the future and is looking for other programmers to work with.",
+    "githubUsername": "SeminaPXL",
+    "twitterUsername": "/",
```

Comment:
If you have no twitter or twitch usernames, please leave black or remove from the file.

Reply:
@whitep4nth3r done